scripts/add-aws-account-with-instance-role.sh

```diff
@@ -1,27 +1,25 @@
 #!/bin/bash
```

Comment:
Some of the scripts are doing non critical setup and are allowed to fail silently. Would like to submit the scripts as is and revisit this on the next update after I've had time to do more testing on the failure paths.